doc/subsystems/networking/qemu_setup.rst

```diff
@@ -160,6 +170,22 @@ To enable IPv4 forwarding the following command should be run as root:
 
    sysctl -w net.ipv4.ip_forward=1
 
+Some applications may also require a DNS server. A number of in-tree samples
+assume by default that the DNS server is available on the host (IP 192.0.2.2),
+which, in modern Linux distributions, usually runs at least a DNS proxy.
+When running with QEMU, it may be required to restart host's DNS, so it served
```

Comment:
``restart the host's DNS, so it can serve requests``

Reply:
done